crates/runtime/src/lib.rs

```diff
@@ -97,14 +117,27 @@ impl Builder {
     }
 }
 
+fn maybe_open_stdio(pipe_path: &Path) -> Option<File> {
```

Comment:
Can you write at least one unit test for this func?

Reply:
Done, but I needed to comment out the other test in the runtime. However, when running make test, it didn't seem like it was running anyway.

I've marked a TODO to re-enable the test.

Reply:
Yes, remove the `test = false` from Cargo.toml